### DISCUSSION TITLE
How to build for manylinux_2_28_x86_64?

### DISCUSSION THREAD

**Discussion by 0xDmtri** [PyO3/maturin#1194] [Q&A] [answered]
I tried ```docker run --rm -v $(pwd):/io quay.io/pypa/manylinux_2_28_x86_64 build  --release```

It all went well until the very end where I got the following error:

Status: Downloaded newer image for quay.io/pypa/manylinux_2_28_x86_64:latest
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
/usr/local/bin/manylinux-entrypoint: line 8: exec: build: not found

Would anyone help please? Many thanks!

**Comment by messense** [comment, marked as answer]
It seems that you are running on arm64 platform, but the `quay.io/pypa/manylinux_2_28_x86_64:latest` is x86_64 only, so docker might try to use QEMU to emulate x86_64 for you.

`docker run --rm -v $(pwd):/io quay.io/pypa/manylinux_2_28_x86_64 build --release` this command is wrong, you are trying to run `build --release` in it which obviously does not work because there isn't an `build` executable in it. For running maturin you should use `docker run --rm -v $(pwd):/io quay.io/pypa/manylinux_2_28_x86_64 maturin build --release`, but you will need to install maturin in it first.

**Reply by 0xDmtri** [reply]
any chance you could suggest how to do that?
